client/extensions/woocommerce/components/dashboard-widget/index.js

```diff
@@ -0,0 +1,140 @@
+/**
+ *
+ * External dependencies
+ *
+ * @format
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Gridicon from 'gridicons';
+import { isUndefined, noop } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Popover from 'components/popover';
+import Tooltip from 'components/tooltip';
+
+class DashboardWidget extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			showDialog: false,
+			showTooltip: false,
+		};
+	}
+
+	hideTooltip = () => {
+		this.setState( { showTooltip: false } );
+	};
+
+	onSettingsPanelClose = () => {
+		this.setState( { showDialog: false } );
+	};
+
+	setSettingsToggle = c => {
+		this.settingsToggle = c;
+	};
+
+	showTooltip = () => {
+		this.setState( { showTooltip: true } );
+	};
+
+	toggleSettingsPanel = () => {
+		this.props.onSettingsClose();
+		this.setState( { showDialog: ! this.state.showDialog } );
+	};
+
+	render() {
+		const {
+			children,
+			className,
+			image,
+			imageFlush,
+			imagePosition,
+			settingsPanel,
+			title,
+			translate,
+			width,
+		} = this.props;
+		const { showDialog, showTooltip } = this.state;
+		const isTopImage = image && 'top' === imagePosition;
+		const imageClassName = image ? `is-${ imagePosition }` : null;
+		const widthClassName = `is-${ width }-width`;
+		const hasSettingsPanel = ! isUndefined( settingsPanel );
+
+		const classes = classNames( 'dashboard-widget', className, imageClassName, widthClassName, {
+			'is-flush-image': imageFlush,
+			'has-settings-panel': hasSettingsPanel,
+		} );
+
+		// eslint-disable-next-line wpcalypso/jsx-classname-namespace
+		const imageComponent = <img className="dashboard-widget__image" src={ image } />;
+
+		return (
+			<Card className={ classes }>
+				<Gridicon
+					className="dashboard-widget__settings-toggle"
+					icon="cog"
+					onClick={ this.toggleSettingsPanel }
+					onMouseEnter={ this.showTooltip }
+					onMouseLeave={ this.hideTooltip }
+					ref={ this.setSettingsToggle }
+				/>
+				<Tooltip
+					baseClassName="dashboard-widget__settings-tooltip"
+					context={ this.settingsToggle }
+					isVisible={ showTooltip }
+				>
+					{ translate( 'Settings', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **38** times:
`translate( 'Settings', { context: 'NewDash Page Title'} )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store/dashboard-widget)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?